src/course-search/tests/SearchBox.test.jsx

```diff
@@ -44,4 +50,17 @@ describe('<SearchBox />', () => {
     expect(history).toHaveLength(3);
     expect(history.location.search).toEqual('');
   });
+  test('tracking event when search initiated with trackingName present in context', () => {
+    renderWithSearchContextAndTracking(<SearchBoxBase />, 'aProduct');
+
+    // fill in search input and submit the search
+    fireEvent.change(screen.getByRole('searchbox'), { target: { value: TEST_QUERY } });
+    fireEvent.click(screen.getByText('submit search'));
```

Comment:
The `react-testing-library` docs say we should use `userEffect` instead of `fireEvent`. Why they still provide `fireEvent` is a mystery to me, but let's keep up-to-date with their recommendations.

Reply:
ah yes indeed, https://testing-library.com/docs/ecosystem-user-event , done!